plugins/codemirror/plugin.js

```diff
@@ -13,8 +13,13 @@
 				var textarea = contentsSpace
 					.getDocument()
 					.createElement('textarea');
+
 				contentsSpace.append(textarea);
 
+				var editable = editor.editable(
```

Comment:
I don't think you should assign this to `editable`, seeing as you never reference it anywhere else.

Reply:
I blindly copied an pastad from CKEditor's code without really thinking about it, but you're right it's not used so not needed, will update that.

And I might as well change the commit message to say `fix`

Reply:
Updated in [https://github.com/liferay/liferay-ckeditor/commit/cd360019946b67e733d2046e6813df44040ef3af](https://github.com/liferay/liferay-ckeditor/commit/cd360019946b67e733d2046e6813df44040ef3af), I also changed the "prefix" in of the commit message